### PR TITLE
Update explore-intro.html for Windows system

### PR DESCRIPTION
**Pull request by FreezeIce1**
some option in Windows system cannot execute, so add some operation for Windows system learner.

### CODE REVIEW

File: content/zh-cn/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 1]
```diff
@@ -274,13 +274,21 @@ <h3>在终端中显示应用</h3>
                 <p><code><b>export POD_NAME="$(kubectl get pods -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')"</b></code><br />
                 <code><b>echo Name of the Pod: $POD_NAME</b></code></p>
 
+                <p>在Windows系统中运行以下命令：</p>
+                <p><code><b>$POD_NAME="$(kubectl get pods -o go-template --template '{{range .items}}{{.metadata.name}}{{end}}')"</b></code><br />
+                <code><b>echo Name of the Pod: ${POD_NAME}</b></code></p>
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/kubernetes-basics/explore/explore-intro.html:279]: Please update the English upstream first, then we sync from the English version.

**Review by kumarankit999 — APPROVED** [review]
looks fixed now, if you change according to @tengqm !!

File: content/zh-cn/docs/tutorials/kubernetes-basics/explore/explore-intro.html (hunk 1 above)
Comment (FreezeIce1) [reply content/zh-cn/docs/tutorials/kubernetes-basics/explore/explore-intro.html:279]: Sure, it ready have a PR.

Comment (chenrui333) [reply content/zh-cn/docs/tutorials/kubernetes-basics/explore/explore-intro.html:279]: can you link the en PR in here as well?

And once the english PR got merged, you might want to sync/rebase your PR

Comment (FreezeIce1) [reply content/zh-cn/docs/tutorials/kubernetes-basics/explore/explore-intro.html:279]: Sure, link is here `https://github.com/kubernetes/website/pull/44016`

**Review by windsonsea — COMMENTED** [review]
The en PR #44016 is closed but not merged, and half a year passed. It's time to close this PR. Feel free to re-open it when you have more improvements, thanks.

/close

**Pull request closed without merging**